src/assets.js

```diff
@@ -247,6 +247,6 @@
     },
 
     // expose properties for testing
-    _canUse: canUse,
+    _canUse: canUse
```

Comment:
You could also add debug around this to save just a few more bytes (it's only used in the test scripts). Same for `_frame` and `_last` in gameLoop.js and `_layerOrder` in tileEngine.js

Reply:
Hi straker, sorry I'm already in holiday mode and currently don't have a computer around but just a nice terry pratchet book ;) merry Christmas and thanks for this nice lib ...